### PR TITLE
Fix Snyk vulnerability and bump internal package version

### PR DESCRIPTION
**Pull request by alasdair-stark-amido**
Added a direct dependency to System.Text.Encodings.Web as there is no update available for Microsoft.AspNetCore.Diagnostics that fixes the issue.

#### 📲 What

Added a direct dependency to System.Text.Encodings.Web
Updated Amido.Stacks.Core patch version

#### 🤔 Why

Address vulnerability reported by Snyk

#### 🛠 How

Added direct dependency to vulnerable package

#### 🕵️ How to test

There are no automated tests in the solution so it may be necessary to test an entry-point project that has a dependency on this package

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: src/Amido.Stacks.API/Amido.Stacks.API.csproj [hunk 1]
```diff
@@ -11,11 +11,12 @@
...
-    <PackageReference Include="Amido.Stacks.Core" Version="0.2.11" />
+    <PackageReference Include="Amido.Stacks.Core" Version="0.2.15" />
     <PackageReference Include="Serilog" Version="2.10.0" />
     <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
   </ItemGroup>
 
 </Project>
```
Comment (ElvenSpellmaker) [inline src/Amido.Stacks.API/Amido.Stacks.API.csproj:22]: Could we have a newline at the end of the file please

File: src/Amido.Stacks.API/Amido.Stacks.API.csproj [hunk 2]
```diff
@@ -11,11 +11,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Amido.Stacks.Core" Version="0.2.11" />
+    <PackageReference Include="Amido.Stacks.Core" Version="0.2.15" />
```
Comment (ElvenSpellmaker) [inline src/Amido.Stacks.API/Amido.Stacks.API.csproj:14]: I assume now I merged the core package this needs further updating?

Comment (ElvenSpellmaker) [reply src/Amido.Stacks.API/Amido.Stacks.API.csproj:14]: 0.2.17 is now the latest

Comment (alasdair-stark-amido) [reply src/Amido.Stacks.API/Amido.Stacks.API.csproj]: That's correct. I've pushed that now.

Comment (ElvenSpellmaker) [reply src/Amido.Stacks.API/Amido.Stacks.API.csproj]: Thanks!

**Pull request merged**